run-tests.sh

```diff
@@ -76,21 +76,69 @@ fi
 
 : ${TARGET_OSES:=${TOOLCHAIN_TARGET_OSES-$DEFAULT_OSES}}
 
-if [ -z "$RUN_X86_64" ] && [ -z "$RUN_I686" ]; then
+# This script can be run in a number of configurations:
+# - On Linux, where one possibly can run tests on the local machine with wine
+#   (if available), and possibly also remotely running tests on a different
+#   machine with suitable COPY_*/RUN_* variables (wrapping scp/ssh).
+# - In WSL, running a native Windows toolchain. (We don't detect this case
+#   automatically, but require the caller to set the suitable RUN_ variables.)
+# - In an msys2 shell.
+
+if [ -z "$RUN_X86_64" ] && [ -z "$RUN_I686" ] && [ -z "$RUN_ARMV7" ] && [ -z "$RUN_AARCH64" ]; then
     case $(uname) in
     MINGW*|MSYS*)
```

Comment:
I guess I should include `CYGWIN` here as well?

Reply:
it'd work the same, sure.

Reply:
> it'd work the same, sure.

As of Cygwin 3.6.0, that is. (they didn't backport it to 3.5 like MSYS2 did.